sections/method/preprocessing.tex

```diff
@@ -0,0 +1,138 @@
+\section{Preprocessing}\label{sec:prepro}
+
+In this section, we will give a detailed overview of our method of preprocessing documents.
+Our preprocessing steps were inspired by \cite{quanti}, who do a thorough preprocessing before applying LDA. 
+Our preprocessing consists of several steps, each of which will be described in more detail, along with a running example over part of a real document.
+The preprocessing steps are:
+\begin{enumerate}[label=\alph*]
+	\item Article Extraction
+	\item Initial Document Filtering
+	\item Stopword and Word Frequency Filtering
+	\item Word Database Filtering
+	\item Lemmatization 
+	\item Construct Files
+\end{enumerate}
+
+\begin{table}
+	\begin{tabular}{c|c|c}
+		Preprocessing Step & Documents & Words\\
+		Article Extraction & 63266 & unknown \\ 
```

Comment:
why is this unknown?

Reply:
vi har aldrig tjekket hvor mange unikke ord der var før at vi begyndte at preprocessere.

Reply:
Det syntes jeg at vi skal gøre :D